lib/utils/debuggability.ts

```diff
@@ -0,0 +1,26 @@
+class Debuggability {
+  private static _debugMode: boolean;
+  private static _stepOverAndPause: boolean;
```

Comment:
```suggestion
  static #debugMode: boolean = false;
  static #stepOverAndPause: boolean = false;
```

Reply:
I think we should avoid changing the functionality of the code while annotating. So, while changing to `#` notation seems fine to me here, we should avoid initializing them with `false`, which will definitely impact the functionality of the transpiled JS code.